client/src/components/NewResourceForm.js

```diff
@@ -208,28 +173,32 @@ const NewResourceForm = (props: FormProps) => {
         setTotalSubmitEnabled,
       })}
       <Form.Item label="Recommendation">
-        {getFieldDecorator('recommendation', {})(
+        {getFieldDecorator('recommendation', {
+          initialValue: 'Yes',
+        })(
           <Select defaultValue="Yes">
             <Option value="Yes">Yes</Option>
             <Option value="Maybe">Maybe</Option>
             <Option value="No">No</Option>
           </Select>,
         )}
       </Form.Item>
-      <Form.Item label="Comments">
-        {getFieldDecorator('comments', {})(
-          <TextArea rows={4} placeholder="Comments..." />,
-        )}
-      </Form.Item>
-      <Form.Item label="Internal Notes">
-        {getFieldDecorator('internalNotes', {})(
-          <TextArea rows={4} placeholder="Internal notes..." />,
-        )}
-      </Form.Item>
+      {CommentsFormItem({
```

Comment:
My opinion on this is that the files `NewResourceCommentsForm` and `NewResourceInternalNotesForm` are wrappers for the component `StrListFormItem`, and so to prevent file clutter and make the html rendering more consistent, we can just use `StrListFormItem` directly here, since the only props that are changing are the data, set function, and set submit function. Instead, we can do:

```js
<StrListFormItem
      formName="commentForm"
      label="Comments"
      placeholder="Enter a comment..."
      listOfStrings={comments}
      setListOfStrings={setComments}
      setTotalSubmitEnabled={setTotalSubmitEnabled}
/>
<StrListFormItem
      formName="internalNotesForm"
      label="Internal Notes"
      placeholder="Enter a note..."
      listOfStrings={internalNotes}
      setListOfStrings={setInternalNotes}
      setTotalSubmitEnabled={setTotalSubmitEnabled}
/>
```

We can also do the same for the language selector form that is essentially a wrapper for this component.

I think that either way is ok, whichever you prefer. This just might help clean when we scale for more components and files.